pennylane/ftqc/parametric_midmeasure.py

```diff
@@ -146,9 +353,133 @@ def label(self, decimals=None, base_label=None, cache=None):  # pylint: disable=
         return _label
 
 
+class XMidMeasureMP(ParametricMidMeasureMP):
+    """A subclass of ParametricMidMeasureMP that uses the X measurement basis
+    (angle=0, plane="XY"). For labels and visualizations, this will be represented
+    as a X measurement. It is otherwise identical to the parent class."""
+
+    _shortname = "measure_x"
+
+    # pylint: disable=too-many-arguments
+    def __init__(
+        self,
+        wires: Optional[Wires],
+        reset: Optional[bool] = False,
+        postselect: Optional[int] = None,
+        id: Optional[str] = None,
+    ):
+        super().__init__(
+            wires=Wires(wires), angle=0, plane="XY", reset=reset, postselect=postselect, id=id
+        )
+
+    def _flatten(self):
+        metadata = (
+            ("wires", self.raw_wires),
+            ("reset", self.reset),
+            ("id", self.id),
+        )
+        return (None, None), metadata
+
+    def __repr__(self):
+        """Representation of this class."""
+        return f"{self._shortname}(wires={self.wires.tolist()})"
+
+    def label(self, decimals=None, base_label=None, cache=None):  # pylint: disable=unused-argument
+        r"""How the mid-circuit measurement is represented in diagrams and drawings.
+
+        Args:
+            decimals=None (Int): If ``None``, no parameters are included. Else,
+                how to round the parameters.
+            base_label=None (Iterable[str]): overwrite the non-parameter component of the label.
+                Required to match general call signature. Not used.
+            cache=None (dict): dictionary that carries information between label calls
+                in the same drawing. Required to match general call signature. Not used.
+
+        Returns:
+            str: label to use in drawings
+        """
+        _label = "┤↗ˣ"
+
+        if self.postselect is not None:
+            _label += "₁" if self.postselect == 1 else "₀"
+
+        _label += "├" if not self.reset else "│  │0⟩"
+
+        return _label
+
+    def diagonalizing_gates(self):
+        """Decompose to a diagonalizing gate and a standard MCM in the computational basis"""
+        return [qml.H(self.wires)]
+
+
+class YMidMeasureMP(ParametricMidMeasureMP):
+    """A subclass of ParametricMidMeasureMP that uses the Y measurement basis
+    (angle=pi/2, plane="XY"). For labels and visualizations, this will be represented
+    as a Y measurement. It is otherwise identical to the parent class."""
+
+    _shortname = "measure_y"
+
+    # pylint: disable=too-many-arguments
+    def __init__(
+        self,
+        wires: Optional[Wires],
+        reset: Optional[bool] = False,
+        postselect: Optional[int] = None,
+        id: Optional[str] = None,
+    ):
+        super().__init__(
+            wires=Wires(wires),
+            angle=np.pi / 2,
+            plane="XY",
```

Comment:
Why not use `angle=0, plane="YZ"`? Or are we running under the assumption that that would correspond to a z-basis measurement?